adhocracy-plus/assets/scss/_variables.scss

```diff
@@ -1,10 +1,17 @@
 // subset of bootstrap variables
-$primary: #008dc5 !default;
-$secondary: #00a747 !default;
+$primary: #2d40cc !default; // added from platform.scss
+$primary-other: #008dc5 !default;
+
+$secondary: #90efe4 !default; // added from platform.scss
+$secondary-other: #00a747 !default;
 $info: #2d40cc !default;
 
-$brand-primary: #008dc5 !default;
-$brand-secondary: #2d40cc !default; //same as $brand-info
+$brand-primary: #2d40cc; // added from platform.scss
+$brand-primary-other: #008dc5 !default;
+
+$brand-secondary: #90efe4 !default; // added from platform.scss
+$brand-secondary-other: #2d40cc !default; //same as $brand-info
+
```

Comment:
I overwrote but left in oldones so when things pop up we can still use them, I think I want to talk to Marcelo about out colour scheme anyway